cmd/stack.go

```diff
@@ -0,0 +1,30 @@
+package cmd
```

Comment:
I feel like this might belong in a `stack` package under `/internal/stack/stack.go` or something like that.

Reply:
I moved `Stack` to `internal/stack` in 70cee1a